com.unity.render-pipelines.universal/Runtime/UniversalRendererData.cs

```diff
@@ -272,39 +272,12 @@ private void ReloadAllNullProperties()
 
         void ISerializationCallbackReceiver.OnBeforeSerialize()
         {
-            m_AssetVersion = k_LatestAssetVersion;
+
         }
 
         void ISerializationCallbackReceiver.OnAfterDeserialize()
         {
-            if (m_AssetVersion <= 0)
-            {
-                var anyNonUrpRendererFeatures = false;
-
-                foreach (var feature in m_RendererFeatures)
-                {
-                    try
-                    {
-                        if (feature.GetType().Assembly == typeof(UniversalRendererData).Assembly)
-                        {
-                            continue;
-                        }
-                    }
-                    catch
-                    {
-                        // If we hit any exceptions while poking around assemblies,
-                        // conservatively assume there was a non URP renderer feature.
-                    }
-
-                    anyNonUrpRendererFeatures = true;
-                }
-
-                // Replicate old intermediate texture behaviour in case of any non-URP renderer features,
-                // where we cannot know if they properly declare needed inputs.
-                m_IntermediateTextureMode = anyNonUrpRendererFeatures ? IntermediateTextureMode.Always : IntermediateTextureMode.Auto;
-            }
-
-            m_AssetVersion = k_LatestAssetVersion;
```

Comment:
This one line has to stay though, otherwise the assets won't have the current version serialized

Reply:
makes sense! addressed in [9f3cd39](https://github.com/Unity-Technologies/Graphics/pull/7280/commits/9f3cd3920e3b0c166444600918385210fe423d1f)

Reply:
The line is still missing 😅

Reply:
oops. Fixed in [0236c4a](https://github.com/Unity-Technologies/Graphics/pull/7280/commits/0236c4ac2d2776fffd88a45e22ddc86f38eeda29)